File: Source/Templates/NetCore/GtkSharp.Template.VBNet/content/GtkSharp.Widget.VBNet/Gtk_Widget.vb
Imports System
Imports Gtk
Imports UI = Gtk.Builder.ObjectAttribute

Namespace GtkNamespace
    Public Class Gtk_Widget
        Inherits Box

        Public Sub New (builder as Builder)
            MyBase.New (builder.GetObject("Gtk_Widget").Handle)

            builder.Autoconnect (Me)
        End Sub
        
        Public Sub New ()
            Me.New (new Builder("Gtk_Widget.glade"))
        End Sub
        
    End Class
End Namespace

File: Source/Templates/GtkSharp.Template.CSharp/content/GtkSharp.Widget.CSharp/Gtk_Widget.cs
using System;
using Gtk;
using UI = Gtk.Builder.ObjectAttribute;

namespace GtkNamespace
{
    public class Gtk_Widget : Box
    {
        public Gtk_Widget() : this(new Builder("Gtk_Widget.glade")) { }

        private Gtk_Widget(Builder builder) : base(builder.GetObject("Gtk_Widget").Handle)
        {
            builder.Autoconnect(this);
        }
    }
}
